tests/spec/filer.spec.js

```diff
@@ -45,21 +46,21 @@ describe('Filer', function() {
   });
 
   it('must honor the \'FORMAT\' flag', function(done) {
-    var name = 'local-test';
+    let name = 'local-test';
```

Comment:
Hi @SahibArora  
```suggestion
    const name = 'local-test';
```
I think you can update the name variable to be a const as it is getting reassigned in the code.

Reply:
Hi @pbrahmbhatt3,
Thanks for coming and looking over.
I will do the necessary changes.